src/systemctl/systemctl.c

```diff
@@ -6379,7 +6379,7 @@ static int enable_unit(int argc, char *argv[], void *userdata) {
         if (carries_install_info == 0 && !ignore_carries_install_info)
                 log_notice("The unit files have no installation config (WantedBy=, RequiredBy=, Also=,\n"
                            "Alias= settings in the [Install] section, and DefaultInstance= for template\n"
-                           "units). This means they are not meant to be enabled using systemctl.\n \n"
+                           "units). This means they are not meant to be enabled using systemctl.\n\n"
```

Comment:
enabled 'by' using systemctl?

Reply:
No, this sentence refers to "systemctl enable". I think it's OK.

Reply:
Oh, the whitespace was actually a trick, because log_notice() otherwise removes the empty line...

Reply:
maybe a comment added here why the space is placed here would be an acceptable fix event though the space in itself of course is a hack